src/main/java/LottoApplication.java

```diff
@@ -0,0 +1,15 @@
+public class LottoApplication {
+    public static void main(String[] args) {
+        LottoGame lottoGame = new LottoGame();
+
+        int budget = InputView.inputPrice();
+        Lottos lottos = lottoGame.buy(budget);
+
+        InputView.printBuyList(lottos);
+        LottoNumbers winningNumbers = LottoNumbers.fromIntegerList(InputView.inputWinningNumbers());
+
+        WinningResults winningResults = LottoDraw.draw(lottos, winningNumbers);
```

Comment:
```suggestion
        WinningResults winningResults = lottos.draw(winningNumbers);
```
위와 같이 다수의 로또를 가지는 Lottos에 메시지를 보내 구현하는 것은 어떨까?